docs/developer-mode.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Developer Mode
+layout: default
+nav_order: 9
+---
+
+## Developer Mode
+
+
+{: .warning }
+Entering and exiting Developer mode will **wipe all** user data!
+Backup anything if you have to.
+
+{: .warning }
+"(placeholder text) insert warning for enrolled (school, buisness) chromebooks"
+
+"(placeholder text) insert detailed but breif description of developer mode"
+
+### Enabling Developer Mode
+1. Press `ESC` + `Refresh` (arrow icon) + `Power` at the same time. This should bring your system into recovery mode.
```

Comment:
I believe the key combo for some devices may be different (jsl?)

Reply:
I'm having trouble finding alternate key combos for recovery mode.

Reply:
ok then, leave it as is